src/pkg/packager/interactive.go

```diff
@@ -18,10 +18,13 @@ import (
 	"github.com/zarf-dev/zarf/src/pkg/utils"
 )
 
-func (p *Packager) confirmAction(stage string, warnings []string, sbomViewFiles []string) bool {
+func (p *Packager) confirmAction(stage string, warnings []string, sbomViewFiles []string) (bool, error) {
```

Comment:
Needed to be updated to handle errors from ColorPrintYAML. Alternatively we might be able to just quietly error or debug log these if Packager has access to a logger and they're low impact. Very open to feedback here!

Reply:
Great question! This is low impact, but we're trying to get rid of all the quiet debugs, unless there is a good reason for it. This is a benign example, but there were plenty of places in the code where we continued because it might not cause a problem and that made implementations confusing and harder to read. If `goyaml.Marshall` errors there probably is something pretty bad going on, though 99.999% of the time it will be fine unless there's an actual issue in our code.